src/data/FormData.js

```diff
@@ -161,11 +161,10 @@ const editProfileFormDataRegular = [
 		}
 	},
 	{
-		placeholder: "LinkedIn Profile",
+		placeholder: "LinkedIn Profile (after the in/)",
```

Comment:
There should be another way of specifying this, as a non-technical user might not know what "after the in/" means